scripts/Engineering/EnginX.py

```diff
@@ -0,0 +1,381 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2019 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+from __future__ import (absolute_import, division, print_function)
+import mantid.simpleapi as simple
+import Engineering.EnggUtils as Utils
+import os
+import csv
+from shutil import copy2
+
+
+# take the user, the run_number of the vanadium, and the directory to save to as arguments
+def main(vanadium_run, user, focus_run, **kwargs):
+
+    # Set all values at top,
+    # get whether or not to force a vanadium, default value is false
+    do_van = kwargs.get("force_vanadium", False)
+    # get whether or not to pre-process the run before focusing, default is false
+    do_pre_process = kwargs.get("pre_process_run", False)
+    # get whether or not to force re-create the calibration, default is false
+    do_cal = kwargs.get("force_cal", False)
+    cropped = kwargs.get("crop_type", None)
+    ceria_run = kwargs.get("ceria_run", "241391")
+    crop_name = kwargs.get("crop_name", "cropped")
+    crop_on = kwargs.get("crop_on", None)
+    grouping_file = kwargs.get("grouping_file", None)
+    if crop_on is None and cropped is not None:
+        print("You must enter a value to crop_on if you enter a cropping method")
+        return
+    root = os.path.abspath(os.sep)
+    if root == "/":
+        path = os.path.expanduser("~/EnginX_Mantid")
+    else:
+        path = os.path.join(root, "EnginX_Mantid")
+    directory = kwargs.get("directory", path)
+    user_dir = os.path.join(directory, "User", user)
+    calibration_directory = os.path.join(user_dir, "Calibration")
+    calibration_general = os.path.join(directory, "Calibration")
+    focus_directory = os.path.join(user_dir, "Focus")
+    focus_general = os.path.join(directory, "Focus")
+
+    run(calibration_directory, calibration_general, ceria_run, crop_name, crop_on, cropped, do_cal, do_pre_process,
+        do_van, focus_directory, focus_general, focus_run, grouping_file, kwargs, vanadium_run)
```

Comment:
Pass `kwargs` last and pass it as `**kwargs` and capture it likewise on the other side.